src/MonoMod.Patcher/MonoMod.Patcher.csproj

```diff
@@ -7,7 +7,8 @@
     <Description>General purpose .NET assembly modding "basework". This package contains the core IL patcher and relinker.</Description>
     <PackageTags>$(PackageTags)</PackageTags>
     <Nullable>annotations</Nullable>
-    <TargetFrameworks>net452;netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Should the patcher build for *all* of MonoMod's targets? I figured it was fine if it hit .NET Standard and one Framework and one Core, being that it's a standalone application first.

Reply:
I figured if it was targeting basically the entire spectrum of .NET runtime kinds, it might as well target everything. I'll revert that change though as soon as I push my cleanup commit.

Reply:
I'm not sure what you mean. The patcher itself should support .NET 7 insofar as Cecil (and the patcher's own implementation) does, regardless of what TFM it's targeting. Plus, a .NET 6 application *should* be able to run on the .NET 7 runtime with minimal changes.